memory_test.go

```diff
@@ -213,28 +213,28 @@ func TestMemCheck(t *testing.T) {
 			// len("abc") + len("a") + SizeNumber
 			3 + 1 + SizeNumber,
 		},
-		{
```

Comment:
Gabri is adjusting mem tracking tests in his current work. Since these were adjusted with the early cutoff, I have just commented these out.